.golangci.yml

```diff
@@ -0,0 +1,14 @@
+run:
+  skip-dirs:
+    - abis
```

Comment:
We can also skip `./contracts/bindings` directory, since it is autogenerated and we don't really want to tinker with anything that lies there. Also the `abis` folder could be moved there, because it is very connected with contracts themselves. But I can do that in future PRs, so don't worry about this one.